### PR TITLE
tests: Add docker cp test

### LINKED ISSUE #306: add docker cp test
add integration docker cp test

### PR DESCRIPTION
**Pull request by GabyCT**
Add integration docker cp test

Fixes #306

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by chavafg** [comment]
lgtm

File: integration/docker/cp_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		cmd := exec.Command("rm", "-f", fileName)
+		err := cmd.Run()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Describe("cp with docker", func() {
+		Context("check files after a docker cp", func() {
+			It("has the corresponding files", func() {
+				fileName := "CPFile"
+				cmd := exec.Command("touch", fileName)
```
Comment (devimc) [inline integration/docker/cp_test.go]: you should use ```ioutil.TempFile``` to generate a file and then use ```path.Base``` to get the base name

Comment (devimc) [inline integration/docker/cp_test.go]: also do not forget to remove this file

File: integration/docker/cp_test.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+				err := cmd.Run()
+				Expect(err).ToNot(HaveOccurred())
+				args = []string{"cp", fileName, id + ":/root/"}
+				runDockerCommand(0, args...)
+				args = []string{"exec", id, "sh", "-c", "ls /root/"}
+				stdout := runDockerCommand(0, args...)
+				Expect(stdout).To(ContainSubstring(fileName))
```
Comment (devimc) [inline integration/docker/cp_test.go]: how about you check the integrity of the file?

**Comment by GabyCT** [comment]
@devimc changes applied

**Pull request merged**